CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+# cuML 0.15.0 (Date TBD)
```

Comment:
You either branched from the wrong branch or are targeting the wrong branch.

Reply:
Indeed, I don't believe branch-0.15 had been created yet when I opened this PR. Fixed